docs/_summary.md

```diff
@@ -26,8 +26,8 @@
 
 * CLI
     * [Overview](cli.md)
-    * [Configuration](cli_configuration.md)
-    * [Commands](cli_commands.md)
+    * [Configuration](cli_dev_configuration.md)
+    * [Commands](cli.md#cli-commands)
```

Comment:
Adding `:id=` to the following lines in `docs/cli.md` will help translators:

```
# CLI Commands :id=cli-comands
```

Reply:
Well spotted; will fix.